pyomo/contrib/doe/fim_doe_tutorial.ipynb

```diff
@@ -0,0 +1,1813 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "print('======Result summary======')\n",
+    "print('This optimization is solved with status:', optimize_result.status)\n",
+    "print('The result FIM is:', optimize_result.FIM)\n",
+    "print('Four design criteria log10() value:')\n",
+    "print('A-optimality:', np.log10(optimize_result.trace))\n",
+    "print('D-optimality:', np.log10(optimize_result.det))\n",
+    "print('E-optimality:', np.log10(optimize_result.min_eig))\n",
+    "print('Modified E-optimality:', np.log10(optimize_result.cond))\n",
+    "\n",
+    "t_list = []\n",
+    "for t in optimize_result.model.t:\n",
+    "    t_list.append(t)\n",
+    "\n",
+    "T_list = []\n",
+    "for i in t_list:\n",
+    "    T_list.append(pyo.value(optimize_result.model.T[i]))\n",
+    "    \n",
+    "si=16\n",
+    "plt.rc('axes', titlesize=si)\n",
+    "plt.rc('axes', labelsize=si)\n",
+    "plt.rc('xtick', labelsize=si)\n",
+    "plt.rc('ytick', labelsize=si)\n",
+    "plt.rc('legend', fontsize=12)\n",
+    "plt.plot(t_list, T_list, 'b', linewidth=2)\n",
+    "#plt.scatter(t_list, T_list, 'b')\n",
+    "plt.ylabel('T [$K$]')\n",
+    "plt.xlabel('Time [$h$]')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Method: Exploratory analysis (Enumeration)\n",
+    "\n",
+    "This method conducts exploratory analysis by enumeration. \n",
+    "It allows a user to define any number (dimensions) of design variables.\n",
+    "Heatmaps can be drawn by two design variables, fixing other design variables; \n",
+    "1D curve can be drawn by one design variable, fixing other design variables."
```

Comment:
You can add this to the list of future tasks for after this PR but I think it would be helpful in this notebook and the online documentation if you provided a brief summary of how to interpret the plots generated using Pyomo.DoE. The text here describes programmatically how Heatmaps and 1D curves are generated but not how to interpret/use them.

Reply:
This is added to both the notebook and the documentation.